src/Augurk.CommandLine/Commands/PublishCommand.cs

```diff
@@ -113,6 +113,11 @@ private int PublishFeatureFiles()
                             return -1;
                         }
                     }
+                    catch (InvalidOperationException)
+                    {
+                        Console.Error.WriteLine($"Unable to parse feature file '{featureFile}'. Possible causes are that the feature file is commented out entirely.");
```

Comment:
Should we make this assumption? Or should we just mention the file does not contain any Gherkin content?

Reply:
;) Fair enough